app/models/operatingsystem.rb

```diff
@@ -23,7 +23,7 @@ class Operatingsystem < ApplicationRecord
   accepts_nested_attributes_for :os_default_templates, :allow_destroy => true,
     :reject_if => :reject_empty_provisioning_template
 
-  validates :major, :numericality => {:greater_than_or_equal_to => 0}, :presence => { :message => N_("Operating System version is required") }
+  validates :major, :presence => { :message => N_("Operating System version is required") }
```

Comment:
Removing the validation completely reduces already quite poor user experience of creating new OS.
While I agree we don't need to enforce both the `major` and `minor` to be numbers, can we keep `major` as number and validate minor with regex?

Reply:
I can do that.